pkg/crc/machine/machine.go

```diff
@@ -382,7 +382,21 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	}
 	result.KubeletStarted = kubeletStarted
 
-	time.Sleep(time.Minute * 3)
+	if err := cluster.StartKubeApiserverOperator(ocConfig); err != nil {
+		result.Error = err.Error()
+		return *result, errors.New(err.Error())
+	}
+
+	if err := cluster.WaitforRequestHeaderClientCaFile(ocConfig); err != nil {
```

Comment:
With the addition of that method, we don't strictly need the whole "enable required deployments" code, we could limit ourselves to restarting `openshift-apiserver` when `requestheader-client-ca-file` changes.

Reply:
Yes, we can do that also but I do find this bit more appealing since it provide us more control over crc side then snc side (which is a long process to test simple changes) . cc @gbraad wdyt?